tests/cloud_test.py

```
@@ -790,3 +791,103 @@ async def test_mideaaircloud_download_lua(self) -> None:
         assert cloud is not None
         with pytest.raises(NotImplementedError), TemporaryDirectory() as tmpdir:
             await cloud.download_lua(tmpdir, 10, "00000000", "0xAC", "0010")
+
+
+class ToshibaIOLifeTest(IsolatedAsyncioTestCase):
+    """ToshibaIOLife cloud test case."""
+
+    responses: ClassVar[dict[str, bytes]] = {}
+
+    APP_KEY = "00000000000000000000000000000000"
+    ACCESS_TOKEN = "e4ddcc22d5ccc270e3b1c876df7c9a8d0a31b22810d0e532c2dd921bb9e0389f"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "ACCESS_TOKEN"

(S105)